src/coreclr/src/vm/jitinterface.cpp

```diff
@@ -12260,6 +12258,8 @@ CorJitResult invokeCompileMethodHelper(EEJitManager *jitMgr,
 #endif
        )
     {
+        CORJIT_FLAGS altJitFlags = jitFlags;
+        altJitFlags.Set(CORJIT_FLAGS::CORJIT_FLAG_ALT_JIT);
```

Comment:
`altJitFlags` is never used. The JIT is going to call back to get the flags. You need:
```
comp->setJitFlags(altJitFlags);
```